server/sonar-server/src/main/java/org/sonar/server/es/DefaultIndexSettings.java

```diff
@@ -45,6 +45,8 @@
   public static final String INDEX = "index";
   public static final String ANALYZED = "analyzed";
   public static final String STRING = "string";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "string" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)